docker/reference/reference_test.go

```diff
@@ -639,7 +639,7 @@ func TestParseNamed(t *testing.T) {
 			failf("error parsing name: %s", err)
 			continue
 		} else if err == nil && testcase.err != nil {
-			failf("parsing succeded: expected error %v", testcase.err)
+			failf("parsing succeeded: expected error %v", testcase.err)
```

Comment:
(This comes from the same repo, but was fixed the same way upstream, so please don’t revert this part.)